pages/multi-token-collections/index.tsx

```diff
@@ -138,13 +142,11 @@ const MultiTokenCollectionList = () => {
                         </NextLink>
                       </td>
                       <td className={styles.addr} title={item.account.eth_address}>
-                        <HashLink label={item.account.eth_address} href={`/account/${item.account.eth_address}`} />
-                        <span className="tooltip" data-tooltip={item.account.eth_address}>
-                          <HashLink
-                            label={`${item.account.eth_address.slice(0, 8)}...${item.account.eth_address.slice(-8)}`}
-                            href={`/account/${item.account.eth_address}`}
-                          />
-                        </span>
+                        {domain ? (
+                          <Address address={item.account.eth_address} domain={domain} />
+                        ) : (
+                          <HashLink label={item.account.eth_address} href={`/account/${item.account.eth_address}`} />
+                        )}
```

Comment:
There were two addresses for pc layout and mobile layout, respectively.
Now there's only one, does it work for mobile layout?

Reply:
Fixed.